ecom_test/tests/backend_tests/products/test_update_regular_price_updates_price.py

```diff
@@ -0,0 +1,74 @@
+
+import pytest
+from ecom_test.src_file.api_helpers.ProductsAPIHelpers import ProductsAPIHelper
+from ecom_test.src_file.utilities.generic_utilities import generate_random_string
+import random
+
+pytestmark = [pytest.mark.products, pytest.mark.regression]
+
+
+@pytest.mark.tcid36
+def test_update_regular_price_updates_price():
```

Comment:
Add a 'doc string' to the function. Explaining what the function is testing.
May be do research if doc string is common/expected in tests.

Reply:
WIP